CHANGES.txt

```diff
@@ -2,6 +2,11 @@
 Changes for Crate-Java-Testing
 ==============================
 
+2024-12-20 0.12.0
+=================
```

Comment:
Let's add the release data + version while we make a new release :) 

```suggestion
Unreleased
==========
```

Reply:
done